.github/workflows/test.yml

```diff
@@ -78,27 +78,52 @@ jobs:
             - name: Install dependencies
               run: yarn --frozen-lockfile
 
-            - name: Generate docs URL
-              id: docsURL
-              uses: actions/github-script@v6
-              with:
-                  script: |
-                      const { buildPreviewURLComment } = await import('${{ github.workspace }}/tasks/build-preview-urls-comment.js');
-                      const body = buildPreviewURLComment(process.env.GITHUB_HEAD_REF);
-                      const urlRegex = /(https?:\/\/[^\s]+)/g;
-
-                      const docsURL = body.match(urlRegex)?.[0];
-                      core.setOutput('docsURL', docsURL);
+            - name: Extract branch name
+              shell: bash
+              run: echo "branch=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" >> $GITHUB_OUTPUT
```

Comment:
This still feels like too much for for GitHub to be able to tell us the branch, but it's less than reaching into an unrelated script.